### DISCUSSION TITLE
Portrait orientation

### DISCUSSION THREAD

**Discussion by smallfly** [NVlabs/instant-ngp#445] [Q&A]
Hello,

Is there something to know/do when using photos that have been taken in the portrait orientation? 

I can see that the fox dataset is in portrait and works great, but if I use a dataset of such photos (taken with a Canon mirrorless) when I run the model in the tested application my view is not correctly oriented.

Is that linked to the meta data in the photos?

Thank you,

![Tested](https://smallfly.com/nerf/Tested.PNG)

**Comment by aaronbshoemaker** [comment]
You could batch rotate the images and try again, but you are probably going to have to mess with the transform_matrix in the colmap2nerf.py would be my guess. Don't know if you can do it from the gui or not.

**Reply by smallfly** [reply]
Thanks for the reply.

But again the fox data set is made of photos in portrait and it's working.  The only difference I see is that the fox photos do not seem to have EXIF data.

**Reply by aaronbshoemaker** [reply]
Canon has exif problems. I've dealt with them a lot in 360 pano work. Check out exifgui. You can batch change or remove the exif data. The problem is how the program is calculating the y axis I believe.

**Reply by smallfly** [reply]
I just did a test after removing all the EXIF/meta data and I still have the issue. Not sure what I'm missing.

**Reply by smallfly** [reply]
I made it work - I had to remove the EXIF data and rotate the actual images. I have been using ImageMagick to do all this pre-processing.

for example:
`magick mogrify -strip -resize 1920 -rotate -90 *.jpg`

**Reply by jonstephens85** [reply]
@smallfly I fixed this issue on my end by re-exporting all of the images as new jpegs using a batch process in Photoshop. I didn't have to rotate the images. I also ran into this issue with any iPhone images.
